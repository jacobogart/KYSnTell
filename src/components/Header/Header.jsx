import React from 'react'

export const Header = (props) => {
  const header = props.location.pathname.split('/')[1].toUpperCase();
  return (
    <header>
      <h1>{header}</h1>
    </header>
  )
}
