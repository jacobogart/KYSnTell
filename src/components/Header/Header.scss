header {
  position: absolute;
  top: 0px;
  text-align: center;
  width: 100vw;
  color: white;
  letter-spacing: 5px;
  h1 {
    font-size: 100px;
    margin: 0;
  }
}

