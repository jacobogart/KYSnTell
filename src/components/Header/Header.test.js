import React from 'react';
import { shallow } from 'enzyme';
import { Header } from './Header';

describe('Header', () => {
  const mockRouterProps = { pathname: '/tell'}
  it('should match snapshot', () => {
    const wrapper = shallow(<Header location={mockRouterProps}/>)
    expect(wrapper).toMatchSnapshot();
  });
})