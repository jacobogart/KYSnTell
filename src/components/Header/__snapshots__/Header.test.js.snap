// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header>
  <Link
    to="/"
  >
    <h1>
      TELL
    </h1>
  </Link>
</header>
`;
